<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Unwrap 360ยบ Images</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="ImageArea" name="sourceImage"/>
    </item>
    <item>
     <widget class="QFrame" name="buttonFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="loadImageButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="zoomInButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="zoomOutButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="settingsButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="goBackButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="processButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="saveImageButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="cancelButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="fullScreenButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_ChooseImage"/>
    <addaction name="action_Settings"/>
    <addaction name="action_Unwrap"/>
    <addaction name="action_SaveUnrappedImage"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <action name="action_ChooseImage">
   <property name="text">
    <string>&amp;Load 360ยบ</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Se&amp;ttings</string>
   </property>
  </action>
  <action name="action_SaveUnrappedImage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save Unwrapped</string>
   </property>
  </action>
  <action name="action_Unwrap">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Unwrap</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageArea</class>
   <extends>QGraphicsView</extends>
   <header>imagearea.h</header>
   <slots>
    <slot>zoomIn()</slot>
    <slot>zoomOut()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_ChooseImage</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Settings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_SaveUnrappedImage</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveResultImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fullScreenButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>enterFullScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settingsButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>processButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>processSourceImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadImageButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveImageButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveResultImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zoomInButton</sender>
   <signal>clicked()</signal>
   <receiver>sourceImage</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zoomOutButton</sender>
   <signal>clicked()</signal>
   <receiver>sourceImage</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cancelProcessing()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>goBackButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setupSourceImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Unwrap</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>processSourceImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadImage()</slot>
  <slot>showSettings()</slot>
  <slot>processSourceImage()</slot>
  <slot>enterFullScreen()</slot>
  <slot>saveResultImage()</slot>
  <slot>cancelProcessing()</slot>
  <slot>setupSourceImage()</slot>
 </slots>
</ui>
